Library/Formula/fish.rb

```diff
@@ -15,26 +15,19 @@ class Fish < Formula
     url "https://github.com/fish-shell/fish-shell.git", :shallow => false
 
     depends_on "autoconf" => :build
-    # Indeed, the head build always builds documentation
     depends_on "doxygen" => :build
```

Comment:
This comment is incorrect; it _will build and install_ without docs. But we still want to build them.